build_tools/i18n/crowdin.py

```diff
@@ -0,0 +1,396 @@
+# -*- coding: utf-8 -*-
```

Comment:
This looks like a tool that should be open sourced ;)

Reply:
see note below - https://github.com/learningequality/kolibri/pull/4254#issuecomment-427394100